extensions/amp-consent/0.1/cmps.js

```diff
@@ -28,12 +28,12 @@ import {getMode} from '../../../src/mode';
 export const CMP_CONFIG = {};
 
 if (getMode().test || getMode().localDev) {
-  CMP_CONFIG['_ping_'] = {
-    'consentInstanceId': '_ping_',
-    'checkConsentHref': '/get-consent-v1',
-    'promptUISrc': '/examples/amp-consent/diy-consent.html',
-  };
 }
+CMP_CONFIG['_ping_'] = {
```

Comment:
can we revert the change here? we don't want to ship the fake cmp

Reply:
Need to find a way to work around this, as the e2e framework doesn't pick up the `ping` CMP b/c it's not in dev mode. Any ideas?